keep/api/tasks/process_event_task.py

```diff
@@ -115,15 +116,21 @@ def __save_to_db(
                         tz=datetime.timezone.utc
                     ).isoformat()
 
-            alert = Alert(
-                tenant_id=tenant_id,
-                provider_type=(
+            alert_args = {
+                "tenant_id": tenant_id,
+                "provider_type": (
                     provider_type if provider_type else formatted_event.source[0]
                 ),
-                event=formatted_event.dict(),
-                provider_id=provider_id,
-                fingerprint=formatted_event.fingerprint,
-                alert_hash=formatted_event.alert_hash,
+                "event": formatted_event.dict(),
+                "provider_id": provider_id,
+                "fingerprint": formatted_event.fingerprint,
+                "alert_hash": formatted_event.alert_hash,
+            }
+            if timestamp_forced is not None:
+                alert_args['timestamp'] = timestamp_forced
```

Comment:
Strangely, it's trying to record None to the db if argument is present, so had to make such a change.

Reply:
what does timestamp_forced mean? when does it happen? not sure I understand why this is needed

Reply:
When we ingest alerts from the dump, we need to enforce the timestamp not to be now(), but to be some time in the past